server/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog (weaverbird python package)
 
+## Unreleased
+
+- PyPika: SQLTranslator has now an attribute for a custom `ValueWrapper`
+  defined as `VALUE_WRAPPER_CLS`.
+- PyPika: Google Big Query translator now use a custom `VALUE_WRAPPER_CLS` for
+  escaping single quote with a slash `\` rather than duplicate it.
```

Comment:
End users do not care about the implem... How about something like "Google Big Query translator now escapes single quotes in string literals with a backslash (`\`) rather than two double quotes (`''`)" ?